packages/allow-scripts/test/utils.js

```diff
@@ -0,0 +1,12 @@
+const { platform } = require('node:os')
+
+const isWindows = platform() === 'win32'
+
+// https://github.com/nodejs/node/issues/38490
+const fixWindowsExecPath = (p) =>
```

Comment:
(should) It'd be harder to forget to use it if it was a general function for compatibility fixes that is always called on paths in allow-scripts test and applies the fixes conditionally inside instead of being applied conditionally in the setup.

Reply:
fc78da758f8d1cfc07d447474bba9ea1a706dc3e

Reply:
p --> path

Reply:
I don't like using `path` specifically as a local var due to common usage of it as reference to `path` module. I went with the (uglier, but hopefully addressing the motivation behind not wanting `p`) `inPath` as a compromise here.